packages/assets/README.md

```diff
@@ -1,4 +1,40 @@
-## Documentation
+# @synergy-design-system/assets
+
+This package provides assets like:
+
+- **logos:** the variants of the SICK brand logo
```

Comment:
isthe definition valid for the future too? I could imagine adding other logos as well. so maybe we should be more generic like (such as SICK brand logo)